src/features/enrollments/components/StudentEnrollmentsPage/index.jsx

```diff
@@ -123,6 +162,38 @@ const StudentEnrollmentsPage = () => {
     dispatch(fetchEligibleCourses());
   }, [dispatch, sortBy, filters, pageTab]);
 
+  const modalBody = isExtendAction ? (
+    <Form.Group controlId="extendDate">
+      <Form.Label>Fill new date</Form.Label>
+      <Form.Control
+        name="date"
+        type="date"
+        value={extendDate}
+        onChange={(e) => setExtendDate(e.target.value)}
+        isInvalid={error}
+      />
+      {
+        error && (
+          <Form.Control.Feedback type="invalid">
+            {error}
+          </Form.Control.Feedback>
+        )
+      }
+    </Form.Group>
+  ) : (
+    <p>
+      Learner with email <b>{selectedRow.learnerEmail}</b> will be <b>{status}</b>{' '}
+      {status === 'enrolled' ? 'to' : 'from'} <b>{selectedRow.ccxName}</b> course.
+    </p>
+  );
```

Comment:
Separe the definition of each content to be clearer, I don't mind if you define that in the same file but the important thing is that it is easier to understand.
```
const renderModalBody = () => {
  if (isExtendAction) return <ExtendDateForm />;
  return <StatusChangeMessage />;
};
```